port/raspberrypi/rp2xxx/src/hal/pins.zig

```diff
@@ -262,82 +459,256 @@ pub const Function = enum {
     }
 };
 
-fn all() [30]u1 {
-    var ret: [30]u1 = undefined;
-    for (&ret) |*elem|
-        elem.* = 1;
+fn all() [48]u1 {
```

Comment:
have you consider having the type of the return here be comptime is has_rp2350b? could still use splat in that case